api/src/main/java/org/openmrs/module/aijar/AijarActivator.java

```diff
@@ -146,6 +146,12 @@ public void started() {
             healthCenter.setName(administrationService.getGlobalProperty(AijarConstants.GP_HEALTH_CENTER_NAME));
             locationService.saveLocation(healthCenter);
 
+            String flagstatus = administrationService.getGlobalProperty("ugandaemr.patientflags.disabledFlags");
+
+            if (flagstatus != null) {
+                administrationService.executeSQL("update patientflags_flag set enabled=0 where name in (" + flagstatus.trim() + ")", false);
```

Comment:
@slubwama You need to break up the comma delimited list, then add a single quote around each flag name, otherwise this statement will match across the whole string